src/stream.ts

```diff
@@ -59,7 +65,6 @@ export class OutboundStream {
     this.closeController.abort()
     // similar to pushable.end() but clear the internal buffer
     await this.pushable.return()
```

Comment:
I don't think this is necessary since the `pushable` is only referenced by this stream class so everything will be garbage collected shortly after the `close` method is called on the `OutboundStream` instance.

```suggestion
```